__tests__/components/AmountInput.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { getComponent } from '@MOCKS/Components'
+import mosaicStore from '@/store/Mosaic'
+//@ts-ignore
+import AmountInput from '@/components/AmountInput/AmountInput.vue'
+// @ts-ignore
+import ErrorTooltip from '@/components/ErrorTooltip/ErrorTooltip.vue'
+import { ValidationProvider } from 'vee-validate'
+import { Tooltip } from 'view-design'
+import { mosaicsMock } from '@MOCKS/mosaics'
+const wrapper = getComponent(
+  AmountInput,
+  { mosaic: mosaicStore },
+  { mosaics: mosaicsMock },
+  { value: 0, mosaicHex: '519FC24B9223E0B4' },
+  { ErrorTooltip, ValidationProvider, Tooltip },
+)
+const vm = wrapper.vm as AmountInput
+describe('AmountInput', () => {
+  describe('the property amount of validationRules should be', () => {
+    test(`excluded:""|is_not:0|min_value:0|maxDecimals:6|max_value:9000000000000000`, () => {
+      wrapper.setProps({ mosaicHex: '519FC24B9223E0B4' })
+      expect(vm.validationRules.amount).toBe(
+        `excluded:""|is_not:0|min_value:0|maxDecimals:6|max_value:9000000000000000`,
+      )
+    })
```

Comment:
why are you testing _validation rules format_ .... any change to validation rules and you will have to modify tests.

please remove these tests and add **number formatting** tests instead.